### PR TITLE
Implement SIA-R55

### PR DESCRIPTION
**Pull request by Jym77**

### CODE REVIEW

File: packages/alfa-rules/src/sia-r55/rule.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      },
+
+      expectations(target) {
+        // Empty groups have been filtered out already, so we can safely get the
+        // first element
+        const role = Node.from(Iterable.first(target).get(), device).role.get()
+          .name;
+
+        const sameResource = Question.of(
+          "reference-equivalent-resources",
```
Comment (Jym77) [inline packages/alfa-rules/src/sia-r55/rule.ts:65]: I'm not fully sure I should reuse that question URI here 🤔

Comment (kasperisager) [reply packages/alfa-rules/src/sia-r55/rule.ts:65]: Nope, we should use a new one appropriate for this case. Something along the lines of `is-content-equivalent` maybe?

**Review by kasperisager — DISMISSED** [review]

**Review by kasperisager — APPROVED** [review]
👍

**Comment by kasperisager** [comment]
Oh, there are actually some unused imports in the rule: https://github.com/Siteimprove/alfa/security/code-scanning

**Pull request merged**